src/mon/Elector.h

```diff
@@ -56,7 +56,7 @@ class Elector {
    *
    * @remarks This function assumes as a default firing value the duration of
    *	      the monitor's lease interval, and adds to it the value specified
-   *	      in @plus
+   *	      in @e plus
```

Comment:
@dachary it emphasizes the following word in italic font. ideally, doxygen could offer some simply way allowing user to create an anchor for an parameter when we reference it, but unfortunately there is no automatic way to do so at this moment, we need create an anchor using the anchor command. so i replaced the dangling refs with @e.